lib/wallet/nodeclient.js

```diff
@@ -135,13 +135,15 @@ class NodeClient extends AsyncEmitter {
   }
 
   /**
-   * Send a transaction. Do not wait for promise.
+   * Send a transaction. Do wait for promise.
    * @param {TX} tx
+   * @param {Boolean} sendBlocking
    * @returns {Promise}
    */
 
-  async send(tx) {
-    this.node.relay(tx);
+  async send(tx, sendBlocking) {
+      sendBlocking ? await this.node.relay(tx, sendBlocking)
```

Comment:
nit: only use ternaries for values, for control flow use if/else

Reply:
fixed ...